modules/python/init.zsh

```diff
@@ -46,6 +46,20 @@ if (( $? && $+commands[virtualenvwrapper.sh] )); then
   source "$commands[virtualenvwrapper.sh]"
 fi
 
+# Load PIP completion.
+if (( $+commands[pip] )); then
+  cache_file="$(mktemp -d)/pip-cache.zsh"
+
+  if [[ "$commands[pip]" -nt "$cache_file" || ! -s "$cache_file" ]]; then
+    # pip is slow; cache its output. And also support 'pip2', 'pip3' variants
+    pip completion --zsh | sed -e "s|compctl -K _pip_completion pip|& pip2 pip3|" >! "$cache_file" 2> /dev/null
```

Comment:
I'd personally prefer to not do a sed replacement here and just use compctl -K _pip_completion pip2 pip3 after sourcing the cache file... I assume that would still work.